modules/scripts/startup-script/files/startup-script-stdlib-head.sh

```diff
@@ -92,8 +94,10 @@ stdlib::announce_runners_end() {
 	if [ "$recursive_proc" -le "0" ]; then
 		if [ "$exit_code" -ne "0" ]; then
 			wall -n "$ERROR_MESSAGE"
+			echo "$ERROR_MESSAGE" >"$SCRIPT_COMPLETE_FILE"
 		else
 			wall -n "$SUCCESS_MESSAGE"
+			cat /dev/null >"$SCRIPT_COMPLETE_FILE"
```

Comment:
`echo -n "" >...` will be a shell built-in. `cat` is external. I'm sure it's installed nearly everywhere, but better to reduce external dependencies.

`type echo` will tell you what a given shell will use (there is also an echo binary).